.github/workflows/ci.yml

```diff
@@ -107,7 +107,9 @@ jobs:
           NEXUS_USERNAME: ${{ secrets.NEXUS_USERNAME }}
           NEXUS_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
         run: |
-          openssl enc -aes-256-cbc -K "$SECRING_GPG_KEY" -iv "$SECRING_GPG_IV" -in "$ENCRYPTED_GPG_KEY_LOCATION" -out "$GPG_KEY_LOCATION" -d &&
```

Comment:
@diegov what do you think if we make the release job only to be executed after PR is merge in master in this repo?

Reply:
That's a good approach I think, in Travis `release.sh` was called unconditionally, and the repo / branch / pr checks were done within the script, so I kept it the same way. But I think it would be better to let the GH action handle it. 

I will log a story to do this.